bindings/wasm/src/document.rs

```diff
@@ -53,6 +55,10 @@ impl NewDocument {
 #[derive(Clone, Debug, PartialEq)]
 pub struct Document(pub(crate) IotaDocument);
 
+#[wasm_bindgen(inspectable)]
+#[derive(Clone, Debug, PartialEq)]
+pub struct Service(pub(crate) IotaService);
```

Comment:
Eventually the API of this type will expand so might as well start it off in a new file.

This type should also have `toJSON`/`fromJSON` methods so it can be constructed from **some** JS value - You can just copy the ones from `Document` or `Method`